.eslintrc.js

```diff
@@ -7,7 +7,6 @@ module.exports = {
     'prettier',
     'plugin:prettier/recommended',
     'plugin:promise/recommended',
-    'plugin:@cspell/recommended',
```

Comment:
You should remove .cspell.json file too. Or are we keeping it in the CI?

Reply:
@p-fernandez keeping it in CI, just making sure locally our linter will run faster